content/doc/book/installing/index.adoc

```diff
@@ -151,19 +140,8 @@ any operating system or platform that supports Java.
 
 *Notes:*
 
-* Unlike downloading and running Jenkins with Blue Ocean in Docker
```

Comment:
unrelated change?

Reply:
yes related as was referencing the `jenkins/blueocean` installation method.

Reply:
agreed